### PR TITLE
Add Azure deploy section

### PR DESCRIPTION
**Pull request by ghogen**

### CODE REVIEW

**Comment by ghogen** [comment]
@gregvanl 
I think this is ready, but would like to check the rendered version.

File: docs/containers/quickstart-python.md [hunk 1]
```diff
@@ -150,14 +154,73 @@ The Docker Explorer provides an interactive experience to examine and manage Doc
 
 1. The output will be displayed in the terminal.
 
+## Build the image in Azure
+
+You can use the command **Azure Container Registry: Build Image in Azure** to build an image that you can then deploy to Azure App Service.
+
+1. Install the [Azure account extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account). Open the **Command Palette** (`kb(workbench.action.showCommands)`) and search for the command **Azure: Sign In**. If you don't have an Azure account, you can sign up for a [free trial](https://azure.microsoft.com/en-us/free/?utm_source=campaign&utm_campaign=vscode-azure-account&mktingSource=vscode-azure-account).
```
Comment (gregvanl) [inline docs/containers/quickstart-python.md:161]: Can you remove the locale from the URL?

File: docs/containers/quickstart-python.md [hunk 2]
```diff
@@ -150,14 +154,73 @@ The Docker Explorer provides an interactive experience to examine and manage Doc
...
 
+- [Learn about using Docker Compose](/docs/containers/docker-compose.md)
 - [Debug with Docker Compose](/docs/containers/docker-compose.md#python)
 - [Customize how you debug Python apps in a container](/docs/containers/debug-python.md)
 - [Customize your Docker build and run tasks](/docs/containers/reference.md)
-- [Push your image to a container registry](/docs/containers/quickstart-container-registries.md#push-an-image-to-a-container-registry)
 - [Push your Django image to an Azure Container Registry](/docs/containers/tutorial-django-push-to-registry.md)
-- [Deploy a containerized app to Azure App Service](https://learn.microsoft.com/azure/developer/python/tutorial-deploy-containers-01)
-- [Learn about using Docker Compose](/docs/containers/docker-compose.md)
+- [Deploy to Azure Container Apps](https://learn.microsoft.com/en-us/azure/container-apps/deploy-visual-studio-code)
```
Comment (gregvanl) [inline docs/containers/quickstart-python.md:226]: Remove en-us locale

File: docs/containers/quickstart-python.md [hunk 3]
```diff
@@ -150,14 +154,73 @@ The Docker Explorer provides an interactive experience to examine and manage Doc
...
+
+   When warned that this will overwrite the previous deployment, choose **Deploy** to confirm.
+
+   This might take a few minutes; you can monitor progress in the terminal window. When it finishes, a button with access to the site is given.
+
+   ![Browse website button](images/app-service/browse-website-button.png)
+
+   Use the button and verify that your change is reflected on the site.
+
+Congratulations, you've used Python in VS Code to create an deploy a web site that's hosted in the cloud and live on the Internet!
```
Comment (gregvanl) [inline docs/containers/quickstart-python.md:211]: Should this be "internet" lower case?
https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=34947

File: docs/containers/quickstart-python.md [hunk 4]
```diff
@@ -9,63 +9,73 @@ MetaDescription: Develop, build, and debug a Python app in a Docker container, u
...
 
-   >**Note:** If **Python: Django** or **Python: Flask** was selected, specify app port for local development. Django defaults to port 8000, while Flask defaults to port 5000; however, any unused port will work.
+   **Django**: The default port 8000.
 
-1. Select either **Yes** or **No** when prompted to include [Docker Compose](/docs/containers/docker-compose.md) files. If you select **Yes**, you will need to [verify the path](/docs/containers/quickstart-python.md#django-apps) to your `wsgi.py` file in the `Dockerfile` to run the **Compose Up** command successfully. Compose is typically used when running multiple containers at once.
+   **Flask**: The defaul port is 5000.
 
-1. With all of this information, the Docker extension creates the following files:
+1. When prompted to include Docker Compose, select **No** if you do not want a Docker Compose file. If you select **Yes**, you will need to [verify the path](/docs/containers/quickstart-python.md#django-apps) to your `wsgi.py` file in the `Dockerfile` to run the **Compose Up** command successfully. Compose is typically used when running multiple containers at once.
```
Comment (gregvanl) [inline docs/containers/quickstart-python.md:63]: I don't think "Django Apps" is a header anymore. Will this link work?

**Comment by ghogen** [comment]
I don't know what the message saying I'm dismissing Jack's review means. That's not something I did intentionally.

**Comment by gregvanl** [comment]
If you have a follow up commit, it dismisses any previous review approvals.

**Review by gregvanl — APPROVED** [review]
Looks good!

**Pull request merged**